login.js

```diff
@@ -0,0 +1,22 @@
+const loginButton = document.querySelector(".login");
+
+// 로그인 버튼 활성화 
+function toggleButton() {
+    // 이메일, 비밀번호 유효성 검사 및 값이 있는지 확인
+    if (emailErr.textContent === "" && passwordErr.textContent === "" &&
+        emailInput.value.trim() !== "" && passwordInput.value.trim() !== "") {
+        loginButton.disabled = false;
+        loginButton.classList.add("active");
+    } else {
+        loginButton.disabled = true;
+        loginButton.classList.remove("active");
+    }
+}
+
+// 로그인 폼 제출 이벤트
+document.querySelector(".login-form").addEventListener("submit", function (e) {
+    e.preventDefault();
+    if (!loginButton.disabled) {
+        window.location.href = "/items.html";
+    }
```

Comment:
💊 제안
form이 유효하지 않으면 submit button이 disabled 상태라 제출이 되지 않으므로 관련 코드는 없어도 될 것 같아요~

```suggestion
    window.location.href = "/items.html";
```